src/sentry/api/endpoints/issues_resolved_in_release.py

```diff
@@ -37,13 +39,26 @@ def get(self, request, project, version):
         except Release.DoesNotExist:
             raise ResourceDoesNotExist
 
+        ids_from_GroupResolution = GroupResolution.objects.filter(
+            release=release,
+        ).values_list(
+            'group_id', flat=True
+        )
+
+        ids_from_GroupCommitResolution = GroupCommitResolution.objects.filter(
+            commit_id__in=ReleaseCommit.objects.filter(
```

Comment:
Oh, I see, in the previous version of the code.

Then let's do this:

* Move forward with this as-is
* Follow up with adding an index to this column so we can stop full sequence scanning this table. Fortunately it's a really small table, so it's not a problem today, but as this grows, it'll become more and more of a problem, so might as well address it sooner than later.